bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -2664,4 +2664,87 @@ namespace JsRTApiTest
     {
         JsRTApiTest::RunWithAttributes(JsRTApiTest::JsCreateStringTest);
     }
+
+    void ApiTest_JsSerializeArrayTest(JsRuntimeAttributes /*attributes*/, JsRuntimeHandle /*runtime*/)
+    {
+        LPCSTR raw_script = "(function (){return true;})();";
+        LPCWSTR raw_wscript = L"(function (){return true;})();";
+
+        // JsSerializeScript has could test coverage and can be used as an oracle for JsSerialize
```

Comment:
Small typo `has could`, should be `has good` I think?

Reply:
yes :)

---
In reply to: [177916805](https://github.com/Microsoft/ChakraCore/pull/4896#discussion_r177916805) [](ancestors = 177916805)